inventories/README.md

```diff
@@ -7,82 +7,43 @@
 On SEAPATH, the definition of our setup must be described inside [Ansible inventories](https://docs.ansible.com/ansible/latest/inventory_guide/intro_inventory.html#passing-multiple-inventory-sources).
 You have to define your own inventories files which match with your setup.
 
-In this directory, you can find some commented examples of inventories for the following case:
-* a SEAPATH cluster demo which can be used to easily test SEAPATH with retails machines (use it as a POC, not in production)
-* a SEAPATH standalone hypervisor setup
-* a complex SEAPATH cluster
+## SEAPATH inventories examples
 
-It is possible with Ansible to provides [multiple inventories files](https://docs.ansible.com/ansible/latest/inventory_guide/intro_inventory.html#passing-multiple-inventory-sources), and we recommend using this feature for complex setup. In this directory, we have separated the "complex SEAPATH cluster" into three inventories. We have a dedicated inventory file for:
-* the physical machines
-* the cluster internal network (Open vSwitch topology)
-* the virtual machines
-## SEAPATH Ansible inventories examples description
+In the `examples` directory, you can find some commented and minimal examples of inventories for a standalone and cluster configuration.
 
-All inventories examples are in YAML format. The example inventories are:
-* `seapath_demo_cluster_definition_example.yml`: SEAPATH cluster demo
-* `seapath_standalone_definition_example.yml`: SEAPATH standalone hypervisor setup
-* The complex SEAPATH cluster inventories are:
-	* `seapath_cluster_definition_example.yml`: physical machines
-	* `seapath_ovstopology_definition_example.yml`: cluster internal network (Open vSwitch topology)
-	* `seapath_vm_definition_example.yml`: virtual machines
+It is possible with Ansible to provides [multiple inventories files](https://docs.ansible.com/ansible/latest/inventory_guide/intro_inventory.html#passing-multiple-inventory-sources), and we recommend using this feature for complex setup.
+In this directory, we have separated the SEAPATH inventories into :
+* seapath-cluster.yaml : Describe the physical machines in a cluster
+* seapath-standalone.yaml : Describe one standalone physical machine
+* seapath_ovs.yml : Describe an extended network over the cluster (use only in cluster mode)
+* seapath-vm-deployement.yaml : Describe the variable to deploy a VM, either in standalone or cluster mode
 
-The directory also contains the firewall rules `iptables_rules_example.txt` which are referring in `seapath_cluster_definition_example.yml`.
+All these inventories contain only the minimal variables required to configure and run SEAPATH. For advanced configuration, please refer to the wiki page : TODO put link
 
-The simple SEAPATH cluster inventory require only minor tweak to be adapted. In all other files described, all the options which do not match a realistic case and need to be modified are indicated.
+### Cluster inventory
```

Comment:
Maybe you can change the title to

### seapath-cluster.yaml

or 

### Cluster inventory: seapath-cluster.yaml

To refer to the file.